Refinements to early design review (#988)
w3ctag/design-reviews: .github/ISSUE_TEMPLATE/010-specification-review.md

こんにちは TAG-さん!

I'm requesting a TAG review of {{feature}}.

{{One paragraph summary of the feature, ideally copy-pasted from your Explainer.}}

  - Explainer¹:
  - Specification:
  - WPT Tests:
  - User research:
  - Security and Privacy self-review²:
  - GitHub repo:
  - Primary contacts:
      - $name (@-mention), $organization/s, $role in developing specification
      - {{repeat as necessary, we recommend including group chairs and editors in this list}}
  - Organization/project driving the specification:
  - Multi-stakeholder support³:
    - Chromium comments:
    - Mozilla comments: https://github.com/mozilla/standards-positions/issues/NNN
    - WebKit comments: https://github.com/WebKit/standards-positions/issues/NNN
    - {{...include feedback/review from developers, implementers, civil society, and others}}
  - Status/issue trackers for implementations⁴:

Further details:

  - [ ] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
  - Relevant time constraints or deadlines:
  - The group where the work on this specification is currently being done:

>   - The group where standardization of this work is intended to be done:

Reviewer: I think we should keep the note that this won't always be needed. Perhaps:
```suggestion
  - The group where standardization of this work is intended to be done (if different from the current group):
```

  - Major unresolved issues with or opposition to this specification:
  - This work is being funded by:

You should also know that...

{{Please tell us anything else you think is relevant to this review.}}

------------------------------------------------------------------------------------
CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING

Please preview the issue and check that the links work before submitting.

> Use links to content rather than pasting text into this issue.  Issues are ephemeral and most of the material we are asking for has long term value.

Reviewer: I feel like the paragraph before this and the paragraph after should be next to each other. Maybe this one belongs first?

¹ We require an explainer to give the relevant context for the spec review, even if the spec has some background information. An explainer must address user needs and contain examples of use.  For more information, see our [explanation of how to write a good explainer](https://tag.w3.org/explainers/).

² A Security and Privacy questionnaire helps us understand potential security and privacy issues and mitigations for your design, and can save us asking redundant questions. See https://www.w3.org/TR/security-privacy-questionnaire/.

³ For your own organization, you can simply state the organization's position instead of linking to it.  This includes items on [Mozilla standards-positions](https://github.com/mozilla/standards-positions), and [WebKit standards-positions](https://github.com/WebKit/standards-positions).  Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.

> ⁴ Include a link to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), or trackers for other implementations if those are known to you.

Reviewer: ```suggestion
⁴ Include links to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), and/or trackers for other implementations if those are known to you.
```

Author: ```suggestion
⁴ Include links to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), and trackers for other implementations if those are known to you.
```

A moratorium on "and/or"...
